source/tests/system/nirfmxlte_driver_api_tests.cpp

```diff
@@ -1110,6 +1110,7 @@ TEST_F(NiRFmxLTEDriverApiTests, ULModAccSingleCarrierFromExample_FetchData_DataL
     EXPECT_SUCCESS(session, client::initiate(stub(), session, "", ""));
 
     // Intermittently gives "374603: Unable to synchronize." and outputs nan values
+    mod_acc_fetch_composite_evm_response = client::mod_acc_fetch_composite_evm(stub(), session, "", 10.0);
```

Comment:
Finally looked at this intermittent test and noticed we never actually called this method so the `mod_acc_fetch_composite_evm_response` was just the default and we weren't doing anything to prevent that `3374603: Unable to synchronize` error from showing up.